packages/venia-concept/src/actions/user/asyncActions.js

```diff
@@ -73,9 +73,17 @@ export const createAccount = accountInfo =>
             dispatch(assignGuestCartToCustomer());
         } catch (error) {
             dispatch(actions.createAccountError.receive(error));
+
+            throw error;
         }
     };
 
+export const createAccount = accountInfo => async dispatch => {
+    try {
+        await dispatch(handleCreateAccount(accountInfo));
```

Comment:
Please add a comment explaining why this code swallows errors. I assume that the expected server-side validation errors are handled somewhere else, but we should indicate that in a comment.

Reply:
I have added a comment